src/ragas/dataset_schema.py

```diff
@@ -511,13 +511,17 @@ def upload(self, base_url: str = RAGAS_API_URL, verbose: bool = True) -> str:
             },
             cls=ChainRunEncoder,
         )
-        upload_packet(
+        response = upload_packet(
             path="/alignment/evaluation",
             data=packet,
+            base_url=base_url
         )
 
```

Comment:
not sure if we should return response here - Ideally we want to do all the error checks somewhere common